handlers/providers/packet/packet.go

```diff
@@ -0,0 +1,34 @@
+package packet
+
+import (
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancherio/go-machine-service/handlers/providers"
+	"github.com/rancherio/go-rancher/client"
+)
+
+func init() {
+	packetHandler := &PacketHandler{}
+	if err := providers.RegisterProvider("packet", packetHandler); err != nil {
+		log.Fatal("could not register packet provider")
+	}
+}
+
+type PacketHandler struct {
+}
+
+func (*PacketHandler) HandleCreate(machine *client.Machine, machineDir string) error {
+	return nil
+}
+
+func (*PacketHandler) HandleError(msg string) string {
+	prettyMsg := msg
+	if strings.Contains(prettyMsg, "POST https://api.packet.net/projects/") && strings.Contains(prettyMsg, "404") {
+		prettyMsg = "Invalid project"
+	}
+	if strings.Contains(prettyMsg, "GET https://api.packet.net/facilities: 401") {
```

Comment:
Why did you break up the POST <url> and 404  in the previous one but just grepped for the exact string here?